station-flattening-v0/src/main/java/net/modificationstation/stationapi/mixin/flattening/MixinLevelProperties.java

```diff
@@ -0,0 +1,40 @@
+package net.modificationstation.stationapi.mixin.flattening;
+
+import net.minecraft.level.LevelProperties;
+import net.minecraft.util.io.CompoundTag;
+import net.modificationstation.stationapi.api.registry.Identifier;
+import net.modificationstation.stationapi.impl.level.StationLevelProperties;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(LevelProperties.class)
+public class MixinLevelProperties implements StationLevelProperties {
+	@Unique private static final String DIMENSIONS_KEY = "StAPIDimensions";
```

Comment:
I think this should be changed to be an identifier instead, like `stationapi:dimension`, to match the rest of custom StAPI info in vanilla tags
```suggestion
	@Unique private static final String DIMENSIONS_KEY = Identifier.of(StationAPI.MODID, "dimensions").toString();
```